site/app/controllers/course/CourseRegistrationController.php

```diff
@@ -48,7 +48,7 @@ public function registerCourseUser(string $term, string $course): void {
         $default_section = $this->core->getQueries()->getDefaultRegistrationSection($term, $course);
         $this->core->getUser()->setRegistrationSection($default_section);
         $this->core->getQueries()->insertCourseUser($this->core->getUser(), $term, $course);
-        $instructor_ids = $this->core->getQueries()->getActiveUserIds(true, false, false, false, false);
+        $instructor_ids = $this->core->getQueries()->getActiveUserIds(true, false, false, false, false, term: $term, course: $course);
```

Comment:
I'm generally not a fan of named parameters in PHP.  Doing this means that future changes to the internal implementation details of `getActiveUserIds()` (i.e., the parameter names) will require developers to edit every callsite where this is used.  Instead, I think the variable names `$term` and `$course` are descriptive enough.  I'm open to other opinions on this though.  Thoughts?